.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+      - alpha
+      - beta
```

Comment:
Are we sure we want to limit these branches (given that semantic-release allows for [a wider set of branches](https://semantic-release.gitbook.io/semantic-release/v/beta/usage/configuration#branches) from which to release)?